pkg/imgpkg/bundle/bundle.go

```diff
@@ -31,6 +31,7 @@ type Logger interface {
 	Debugf(msg string, args ...interface{})
 	Tracef(msg string, args ...interface{})
 	Logf(msg string, args ...interface{})
+	UI() goui.UI
```

Comment:
hmm, seems a bit weird for logger to carry ui. seems like we are spilling out internal implementation.